dev/integration_tests/flutter_gallery/pubspec.yaml

```diff
@@ -169,7 +169,6 @@ flutter:
     - packages/flutter_gallery_assets/products/table.png
     - packages/flutter_gallery_assets/products/teaset.png
     - packages/flutter_gallery_assets/products/top.png
-    - packages/flutter_gallery_assets/people/ali.png
```

Comment:
I think this line shouldn't have been here as this file didn't exist. This was erroneously working, because there existed an `ali.png` at `packages/flutter_gallery_assets/people/square/ali.png`. The variant parsing code parsed that as a `square` variant (which eventually gets interpreted as a 1x variant).